src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -100,6 +114,16 @@ const LaunchAnalysisModal = ({
     ]),
     div({ style: { color: colors.danger(), overflowWrap: 'break-word' } }, [
       h(Fragment, wrappableOnPeriods(launchError))
+    ]),
+    div(['Output files will be saved as workspace data in:']),
+    div({ style: { margin: '1rem' } }, [
+      location ? h(Fragment, [span({ style: { marginRight: '0.5rem' } }, [flag]),
+        span({ style: { marginRight: '0.5rem' } }, [regionDescription]),
+        h(InfoBox, [
+          p(['Output (and intermediate) files will be written to your workspace bucket in this region.']),
+          p(['Tasks within your workflow might additionally move or copy the data elsewhere. You should carefully vet the workflows you run if region requirements are a concern.']),
+          p(['Note that metadata about this run will be stored in the US.'])
+        ])]) : 'Loading...'
```

Comment:
What is the purpose of `Loading...` here? Is it like if the region hasn't been instantiated it displays `Loading` till then?

Reply:
Never mind. Please ignore the question. I tested it locally and found out the purpose of `Loading`. It is exactly what I said ^

Reply:
It's so that if the API call to google hasn't replied yet we can still display the rest of the content around it and leave a placeholder for the region information.